nodeJS/apis/api_security.md

```diff
@@ -26,15 +26,15 @@ This strategy, while particularly useful with APIs can be used with a traditiona
 </div>
 
 
-### Additional Resources
+### Additional resources
 This section contains helpful links to other content. It isn't required, so consider it supplemental.
 
 - [This article](https://laptrinhx.com/a-practical-guide-for-jwt-authentication-using-node-js-and-express-917791379/) goes even further in depth than the one that we posted above. It provides a little greater understanding but is harder to follow.
 - [This article](https://medium.com/@paul.allies/stateless-auth-with-express-passport-jwt-7a55ffae0a5c) covers the same thing as the above one, but is a little more concise.  Might be a good reference for when you're setting this up on your own later.
 - Not everyone agrees that JWTs are the best way to store authentication data. [This video](https://www.youtube.com/watch?v=JdGOb7AxUo0) is one argument against using them, and some of the pitfalls that you may encounter with using them.
 
 
-### Knowledge Checks 
+### Knowledge checks 
```

Comment:
```suggestion
### Knowledge check
```

This should not have the `s` but I realise it was already there when you edited. 
You can click the below button `commit suggestion` if you agree :) That way you don't have to change it yourself and will just commit my suggested code directly!